packages/core/admin/admin/src/content-manager/components/DynamicZone/components/DzLabel/index.js

```diff
@@ -33,12 +33,24 @@ const DzLabel = ({ label, labelAction, name, numberOfComponents, required }) =>
           shadow="filterShadow"
           color="neutral500"
         >
-          <Flex>
-            <Typography fontSize={0} lineHeight={0} textColor="neutral600" fontWeight="bold">
-              {intlLabel}&nbsp;({numberOfComponents})
-            </Typography>
-            {required && <Typography textColor="danger600">*</Typography>}
-            {labelAction && <Box paddingLeft={1}>{labelAction}</Box>}
+          <Flex direction="column" justifyContent="center">
+            <Flex maxWidth={pxToRem(356)}>
+              <Typography variant="pi" textColor="neutral600" fontWeight="bold" ellipsis>
+                {intlLabel}&nbsp;
+              </Typography>
+              <Typography variant="pi" textColor="neutral600" fontWeight="bold">
+                ({numberOfComponents})
+              </Typography>
+              {required && <Typography textColor="danger600">*</Typography>}
+              {labelAction && <Box paddingLeft={1}>{labelAction}</Box>}
+            </Flex>
+            {description && (
+              <Box paddingTop={1} maxWidth={pxToRem(356)}>
+                <Typography variant="pi" textColor="neutral600" ellipsis>
+                  {description}
```

Comment:
Can you use `formatMessage` so the description are translatable like the label?

Reply:
Thanks for the review, bth I don't understand why should it be translated since it is the description saved by the user in the configure edit view form? I mean it is not an intl key. Did I miss something?

Reply:
Yes, with the v4 and the customisation it's now possible to add a translation key for the content manager which makes it possible to translate the entire model for the content manager.

This is why you have an `intlLabel` L.22 of the file.


You can test it by adding a key to your app.js that matches the description set in your setting.

Reply:
Oh ok I understand thanks!

Reply:
It is updated!